cli/pull.ml

```diff
@@ -1,4 +1,35 @@
-let run repo () = Duniverse_lib.Dune_cmd.gen_dune_upstream_branches repo
+open Stdune
+open Duniverse_lib
+
+let mark_duniverse_content_as_vendored ~duniverse_dir =
+  let open Result.O in
+  let dune_file = Fpath.(duniverse_dir / "dune") in
+  let content = "(vendored_dirs *)" in
+  Logs.debug (fun l -> l "Writing %s to %a" content Styled_pp.path dune_file);
+  Bos.OS.File.write dune_file content >>= fun () ->
+  Logs.debug (fun l -> l "Successfully wrote %a" Styled_pp.path dune_file);
+  Ok ()
```

Comment:
Perhaps just add this stanza rather than erase an existing `dune` file in duniverse/ ?  There might be other custom stanzas in there.

Reply:
Do you have an example of what one would want in a `duniverse/dune`? Might be worth doing it through `duniverse` directly, eventually via CLI fags.

Since `duniverse/` is generated I think it's okay to overwrite whatever is in there, at least in a first iteration and especially since blindly appending could lead to weird stuff as well.

We could also add a comment saying this file is generated and discourage editing it.

I think while it's delusional to think we'll get it right in this PR straight away and that people won't need to edit the content of the `duniverse/`, it definitely is what we should aim for!

Reply:
I mainly just have local hacks in there at the moment (e.g.  a rule to copy executables into an output/ directory).  Let's go with your suggestion of just adding a comment to mark it as autogenerated... I totally agree with your sentiment that duniverse should "just work" :-)